bindings/node.js/.gitignore

```diff
@@ -0,0 +1,3 @@
+build
+ckzg.node
+ckzg_wrap.cxx
```

Comment:
This is the C++ file produced by [SWIG](https://www.swig.org/). We could check it in to save people the trouble of running SWIG.

Reply:
Is SWIG necessary for this? There are only a few functions - I could probably write the wrapper manually. I would want to look at what SWIG produced first.

In any case checking it in does sound like the right thing to do.

Reply:
I think you're right. I will try again using Node-API directly instead of SWIG.